eng/code-quality-reports/src/main/java/com/azure/tools/revapi/transforms/IgnoredTransitiveChangesTransform.java

```diff
@@ -51,18 +91,24 @@ public TransformationResult tryTransform(@Nullable E oldElement, @Nullable E new
             return TransformationResult.keep();
         }
 
-        if (!newArchive.startsWith("com.azure:azure-core:")
-            && !newArchive.startsWith("com.azure:azure-json:")
-            && !newArchive.startsWith("com.azure:azure-xml:")) {
-            // The difference isn't from the azure-core, azure-json, or azure-xml SDK, keep the current result.
+        boolean shouldKeep = true;
+        for (String ignoredNewArchive : ignoredNewArchives) {
+            if (newArchive.startsWith(ignoredNewArchive)) {
+                shouldKeep = false;
+                break;
+            }
+        }
+
+        if (shouldKeep) {
+            // The difference didn't match any 'ignoredNewArchives', keep the current result.
             return TransformationResult.keep();
         }
 
         // Only transform the Difference if the criticality is an error. If this isn't guarded it results in an
         // infinite transformation loop as RevApi will keep running the transformation pipeline until there are no
         // transformations applied in the pipeline run.
         if (difference.criticality == Criticality.ERROR) {
```

Comment:
are we eating more errors than we intend to here?

Reply:
Nope! We'll only reach this point if we know we're matching an archive that should be ignored. This is just a guard against an edge case in RevApi that's called out in the code comments.